### PR TITLE
fix: Duplicate entries allowed in Ticket-Tag

### LINKED ISSUE #6350: Multiple rows with same value allowed in ticket-tags
**Describe the bug**
It should check if the same ticket-tag already exists for the specified event.

**Expected behavior**
Should throw an error that entry already exists in case it does.

**Additional context**
Working on it.

### PR DESCRIPTION
**Pull request by prateekj117**
Fixes #6350 

#### Short description of what this resolves:
It should check if the same ticket-tag already exists for the specified event.

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

- [X] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: app/api/ticket_tags.py [hunk 1]
```diff
@@ -28,10 +28,24 @@ def before_post(self, args, kwargs, data):
...
             raise ForbiddenException({'source': ''}, 'Co-organizer access is required.')
 
+    def before_create_object(self, data, view_kwargs):
+        """
+        method to check if ticket tag object already exists for an event
+        :param data:
+        :param view_kwargs:
+        :return:
+        """
+        if self.session.query(TicketTag).filter_by(event_id=data['event'], name=data['name'], deleted_at=None).first():
```
[the remark this answers is not in the document]
Comment (prateekj117) [reply app/api/ticket_tags.py]: @mrsaicharan1 Yup, that's why I am using `event_id` as well.

[the remark this answers is not in the document]
Comment (mrsaicharan1) [reply app/api/ticket_tags.py]: @prateekj117 Can't a single event have multiple tags with the same name?

[the remark this answers is not in the document]
Comment (iamareebjamal) [reply app/api/ticket_tags.py]: @mrsaicharan1 What'd be the usecase of that?

[the remark this answers is not in the document]
Comment (mrsaicharan1) [reply app/api/ticket_tags.py]: @iamareebjamal @prateekj117  Okay. Yeah, wouldn't have any advantage. This is fine.

**Comment by prateekj117** [comment]
@iamareebjamal Done.

**Pull request merged**